scripts/projectView.js

```diff
@@ -1,23 +1,26 @@
-projectView = {};
+(function(module){
+  projectView = {};
 
-projectView.handleMainNav = function() {
-  $('.main-nav').on('click', 'li', function(){
-    var $content = $(this).attr('data-content');
-    $('.tab-content').hide();
-    $('.tab-content[id = "'+$content +'"]').fadeIn();
-  });
-        $('.main-nav .tab:first').click();
-};
+  projectView.handleMainNav = function() {
+    $('.main-nav').on('click', 'li', function(){
+      var $content = $(this).attr('data-content');
+      $('.tab-content').hide();
+      $('.tab-content[id = "'+$content +'"]').fadeIn();
+    });
+          $('.main-nav .tab:first').click();
+  };
```

Comment:
You could handle showing the landing tab in pure CSS, but you might want to wait for Monday and routing with page.js to really make a decision here, since that will change how you handle stuff like this.